client/store/playbackManager.ts

```diff
@@ -140,6 +141,32 @@ export const getters: GetterTree<PlaybackManagerState, RootState> = {
         return stream.Type === 'Subtitle';
       });
     }
+  },
+  getCurrentTime: (state) => {
+    return state.currentTime;
+  },
+  getDuration: (state, _getters, _rootState, rootGetters) => {
```

Comment:
This shouldn't be a thing, imo.

You can already get the current item pretty easily, there's no need for getters to get properties of the current item, it's only adding more API surface for no real benefit.